common/app/model/RelatedContent.scala

```diff
@@ -25,8 +25,17 @@ case class RelatedContent (
 
 object RelatedContent {
   def apply(parent: ContentType, response: ItemResponse): RelatedContent = {
-    // It's misleading to use storyPackage here rather than relatedContent. A tidy up should rename this file.
-    val items = response.storyPackage.map { item =>
+    // It's misleading to use storyPackage here rather than relatedContent. A tidy up should rename this object
```

Comment:
I agree this is very misleading. It took me more than it should have to understand how `RelatedContent` works.
I am happy to spend some time and refractor this once this PR is out. @rich-nguyen